aim/tests/unit/test_aim_manager.py

```diff
@@ -242,13 +242,19 @@ def _test_resource_ops(self, resource, test_identity_attributes,
         # Verify successful creation
         r1 = self.mgr.create(self.ctx, res)
         for k, v in creation_attributes.iteritems():
-            self.assertEqual(v, getattr_canonical(r1, k))
+            if isinstance(v, list):
```

Comment:
Are you sure this is needed? We have other objects that use lists (e.g. SecurityGroupRule, EndpointGroup)

Reply:
In my testing, it also works sometimes if I change the elements of that list so I guess we were probably just lucky before. I think this is also because all our UTs probably don't have a long list of SGRs or EPGs to test with. But this change does make sense to me though because 2 lists should be considered equal as long as their elements are all the same even if the order is different.